### PR TITLE
Mcga 8

### PR DESCRIPTION
**Pull request by Taimoorrana1**
Building object can now be retrieved by clicking on the building.

### CODE REVIEW

File: app/src/test/java/com/concordia/mcga/models/BuildingUnitTest.java [hunk 1]
```diff
@@ -0,0 +1,644 @@
...
+        LatLng edge = new LatLng(45.495656, -73.574290);
+        list.add(edge);
+        testBuilding.addEdgeCoordinate(list);
+        PolygonOptions polygonOptions = testBuilding.getPolygonOverlayOptions();
+        assertEquals(0x996d171f, polygonOptions.getFillColor());
+    }
+
+    @Test
+    public void getBuildingWithMarkerTest(){
+        zzf z1 = new zzf() {
```
Comment (arekmano) [inline app/src/test/java/com/concordia/mcga/models/BuildingUnitTest.java]: Robolectric is used to prevent exactly this case

Comment (Taimoorrana1) [inline app/src/test/java/com/concordia/mcga/models/BuildingUnitTest.java]: By using shadow?

Comment (arekmano) [inline app/src/test/java/com/concordia/mcga/models/BuildingUnitTest.java]: yes

**Comment by arekmano** [comment]
Approving changes as correctly writing tests is not a big deal as long as they demonstrate that the functionality works correctly. Only thing to be fixed are the merge conflicts

File: app/src/test/java/com/concordia/mcga/models/CampusTest.java [hunk 2]
```diff
@@ -0,0 +1,603 @@
...
+
+            }
+
+            @Override
+            public boolean isVisible() throws RemoteException {
+                return false;
+            }
+
+            @Override
+            public boolean zzj(zzf zzf) throws RemoteException {
```
Comment (mihaiqc) [inline app/src/test/java/com/concordia/mcga/models/CampusTest.java]: Do we need to test these?

Comment (arekmano) [inline app/src/test/java/com/concordia/mcga/models/CampusTest.java]: Not a test. Its a mock object.

**Pull request merged**